src/app/api/waiting/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export const GET = async (request: NextRequest) => {
+  const { searchParams } = new URL(request.url);
+  const searchCode = searchParams.get('code') || '';
+
+  const response = await fetch(
+    `${process.env.NEXT_PUBLIC_SERVER_ADDRESS}:${process.env.NEXT_PUBLIC_SERVER_PORT}/api/photo-request/status`,
+    {
+      headers: {
+        Authorization: `Bearer ${searchCode}`,
+      },
+    },
+  );
+  const jsonData = await response.json();
+
+  return NextResponse.json(
+    {
+      code: jsonData.code,
+      data: jsonData.data,
+      msg: jsonData.msg,
+    },
```

Comment:
별건 아니지만 은닉해야할 데이터가 있는게 아니라면 API 응답값을 그대로 넣어줘도 괜찮을 것 같아요
```suggestion
    jsonData,
```

Reply:
넵 수정했습니다! 감사합니다~